runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -1353,6 +1358,55 @@ J9::ARM64::PrivateLinkage::copyParametersToHomeLocation(TR::Instruction *cursor)
    return cursor;
    }
 
+/**
+ * A more optimal implementation of this function will be required when GRA is enabled.
+ * See OpenJ9 issue #6657.  Also consider merging with loadStackParametersToLinkageRegisters.
+ */
+TR::Instruction *
+J9::ARM64::PrivateLinkage::copyParametersToHomeLocation(TR::Instruction *cursor, bool parmsHaveBeenStored)
+   {
+   TR::Machine *machine = cg()->machine();
+   TR::ARM64LinkageProperties& properties = getProperties();
+   TR::RealRegister *javaSP = machine->getRealRegister(properties.getStackPointerRegister());       // x20
+
+   TR::ResolvedMethodSymbol *bodySymbol = comp()->getJittedMethodSymbol();
+   ListIterator<TR::ParameterSymbol> parmIterator(&(bodySymbol->getParameterList()));
+   TR::ParameterSymbol *parmCursor;
+
+   // Store to stack all parameters passed in linkage registers
+   //
+   for (parmCursor = parmIterator.getFirst();
+        parmCursor != NULL;
+        parmCursor = parmIterator.getNext())
+      {
+      if (parmCursor->isParmPassedInRegister())
+         {
+         if (!parmsHaveBeenStored)
```

Comment:
Do you need to execute this loop with `parmIterator` when `parmsHaveBeenStored` is true?

Reply:
I think that we will add more logic inside this loop when implementing GRA. Please see x version: https://github.com/eclipse/openj9/blob/99a64d22d23429b6cced01f7757b86cdea0baa9c/runtime/compiler/x/codegen/X86PrivateLinkage.cpp#L252-L337

Reply:
OK, I understand.